addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1,5 +1,6 @@
 odoo.define('web_editor.wysiwyg', function (require) {
 'use strict';
+const dom = require('web.dom');
```

Comment:
There should be a blank line after the 'use strict' stuff, just like before :grimacing: